hackathon_site/hardware/models.py

```diff
@@ -95,3 +96,22 @@ class Incident(models.Model):
 
     def __str__(self):
         return self.id
+
+
+class AnnotatedHardwareManager(models.Manager):
+    def get_queryset(self):
+        return super()
+            .get_queryset()
+            .annotate(
+                quantity_checked_out=Count(
+                    "order_items",
+                    filter=(
+                        Q(order_items__part_returned_health__isnull=True)
+                        & ~Q(order_items__order__status="Cart")
```

Comment:
Once you merge in the develop branch, the "Cart" status is now gone, but we added a new "Cancelled" status which should be excluded here. So this can just be swapped with "Cancelled" instead

Reply:
OK actually, there's a circular dependency between this and #286. Just leave as Cart for now and write tests accordingly, but perhaps leave a `# TODO:` in the code so we remember to fix it later.